code/datums/client_image_group.dm

```diff
@@ -0,0 +1,90 @@
+var/global/list/datum/client_image_group/client_image_groups
+
+/datum/client_image_group
+	var/list/image/images
+	var/list/loc_to_image_lookup
+	var/list/subscribed_mobs
```

Comment:
bit of a misnomer, since it's an associated list of mob -> refcount

Reply:
Refactored towards hopefully more clear variable names.